commit 882bb1851ab12ee3ba1561552faf90850e212052
Author: Michael Tremer <michael.tremer@ipfire.org>
Date:   Sun Mar 11 17:59:34 2012 +0100

    QA: libs-location: Revert checking in /usr/lib(64).
    
    That was not such a good idea because upstream is messing
    too much with library names.

diff --git a/tools/quality-agent.d/003-libs-location b/tools/quality-agent.d/003-libs-location
index 479f170..0ee8237 100755
--- a/tools/quality-agent.d/003-libs-location
+++ b/tools/quality-agent.d/003-libs-location
@@ -25,7 +25,7 @@ function check() {
 	local failed=0
 	local found
 
-	for lib in $(find ${BUILDROOT}/{,usr/}${libdir} -maxdepth 1 -type f -name "lib*.so.*" 2>/dev/null); do
+	for lib in $(find ${BUILDROOT}/${libdir} -maxdepth 1 -type f -name "lib*.so.*" 2>/dev/null); do
 		lib=${lib##*/}
 		lib=${lib%%.so*}
 
